src/components/xy_chart/series/line_series.js

```diff
@@ -3,36 +3,54 @@ import PropTypes from 'prop-types';
 
 import { LineSeries, MarkSeries, AbstractSeries } from 'react-vis';
 
-export class EuiLine extends AbstractSeries {
+export class EuiLineSeries extends AbstractSeries {
   render() {
-    const { 
-      data, 
-      name, 
-      curve, 
-      onClick, 
-      onMarkClick, 
-      hasLineMarks, 
-      lineMarkColor, 
-      lineMarkSize, 
-      color, 
-      ...rest 
+    const {
+      data,
+      name,
+      curve,
+      onClick,
+      onMarkClick,
+      showLine,
```

Comment:
What is the rational behind adding this option vs just removing the series from JSX? If the idea is to allow for just marks, then I think a mark series would make more sense.

Reply:
Good point. I think you are right, having a scatterplot or mark series it's a requirement and make more sense.